docs/linux-intel-gpu-setup.md

```
@@ -100,6 +106,54 @@ sudo apt-get install -y libvulkan1 mesa-vulkan-drivers vulkan-tools
+```bash
+ls /usr/lib/x86_64-linux-gnu/libze_intel_npu.so*   # plugin must exist
+# Use the app's managed Python venv (system python3 won't have openvino):
+~/.local/share/ai-playground/resources/OpenVINO/.venv/bin/python3 -c \
+  "import openvino as ov; print(ov.Core().available_devices)"
+# Expected: ['CPU', 'GPU', 'NPU']
+```
+
+> **Kernel pairing.** The NPU userspace ABI is locked to the kernel `intel_vpu`
+> module. Panther Lake needs **kernel ≥ 6.10**; on older Ubuntu kernels
+> install `linux-generic-hwe-24.04`.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)